telink: tl721x: update west.yml and Kconfig.soc

jinmiaoyu opened the pull request:
- add MICRO_SPEECH_EXTRA_DRIVER in Kconfig.soc
- add tflite-micro-telink in west.yml

jinmiaoyu committed:
telink: tl721x: update west.yml and Kconfig.soc

- add MICRO_SPEECH_EXTRA_DRIVER in Kconfig.soc
- add tflite-micro-telink in west.yml

Signed-off-by: Jinmiao Yu <jinmiao.yu@telink-semi.com>

andriy-bilynskyy commented:
The following west manifest projects have been modified in this Pull Request:

| Name | Old Revision | New Revision | Diff |
| ---- | ------------ | ------------ |------|
| hal_telink | https://github.com/telink-semi/hal_telink/commit/d91dedb76c034b3e89f3942b495b5327c05db5e5 (develop_customer_new_base) | https://github.com/telink-semi/hal_telink/commit/36e84923b4473a8beb2a61502c60282c2b28bd6a (develop) | [telink-semi/hal_telink@d91dedb7..36e84923](https://github.com/telink-semi/hal_telink/compare/d91dedb76c034b3e89f3942b495b5327c05db5e5..36e84923b4473a8beb2a61502c60282c2b28bd6a) |
| tflite-micro-telink | None | c89ed44791d592f4435c7fa924d2edaa0aa0e071 | N/A |

*Note: This message is automatically posted and updated by the Manifest GitHub Action.*

jinmiaoyu committed:
telink: tl721x: added tflite-micro-telink in MAINTAINERS.yml

- added West project: tflite-micro-telink in MAINTAINERS.yml
- replace the hal_telink maintainers

Signed-off-by: Jinmiao Yu <jinmiao.yu@telink-semi.com>
